### PR TITLE
Revolution is started

### PR DESCRIPTION
**Pull request by toprak**
I am planning to make it more detailed and separated as categories.

### CODE REVIEW

**Comment by toprak** [comment]
Do not push it yet, When I complete it then do it bro.

**Comment by anshumanv** [comment]
Cool, thanks! Let's bring this revolution 😆

Lmk when I should review this

**Comment by toprak** [comment]
I will continue to revolution at sunday :D

**Comment by toprak** [comment]
@anshumanv done bro, if I remember something or some idea would show up on my head i will add.

**Comment by toprak** [comment]
Mate, are you alive~~

**Comment by toprak** [comment]
I have an university entrance exam in June 18-19 so I will be afk until the day, hope I will be succesfull  about my dreams. When I return, I will return this revolution again with more detailed and new format also add acknowledges about sites regarding ads etc.

**Comment by anshumanv** [comment]
> Mate, are you alive~~

Yes, will review this today 💯

File: README.md [hunk 1]
```diff
@@ -14,83 +14,230 @@ A curated list of anime streaming/downloading websites.
...
+* [myanimelist.net](https://myanimelist.net)
+* [anilist.co](https://anilist.co)
+* [anisearch.com](https://anisearch.com)
+* [annict.com](https://annict.com)
+* [simkl.com](https://simkl.com)
+* [anime-planet.com](https://anime-planet.com]
+
+### Schedules/Charts
+* [myanimelist.net/anime/season](https://myanimelist.net/anime/season)
+* [anidb.net](htps://anidb.net/anime/season/?do.filter=1) Season chart
```
Comment (anshumanv) [inline README.md:125]:
```suggestion
* [anidb.net/anime/season](htps://anidb.net/anime/season/?do.filter=1) Season chart
```

File: README.md [hunk 2]
```diff
@@ -14,83 +14,230 @@ A curated list of anime streaming/downloading websites.
...
+* [anidb.net](https://anidb.net/anime/schedule/?do.filter=1) Schedule
```
Comment (anshumanv) [inline README.md:126]:
```suggestion
* [anidb.net/anime/schedule](https://anidb.net/anime/schedule/?do.filter=1) Schedule
```

File: README.md [hunk 3]
```diff
@@ -14,83 +14,230 @@ A curated list of anime streaming/downloading websites.
...
+* [senpai.moe](https://www.senpai.moe/)
+* [anisearch.com](https://www.anisearch.com/anime/calendar)
+* [livechart.me](https://livechart.me)
+* [anichart.net](https://anichart.net/)
+
+### Forums/Boards for discussing Anime
+* [4chan.org](https://boards.4channel.org/a/)
+* [reddit.com](reddit.com/r/anime)
```
Comment (anshumanv) [inline README.md:134]:
```suggestion
* [reddit.com/r/anime](reddit.com/r/anime)
```

**Review by anshumanv — COMMENTED** [review]
Looks amazing! Some suggestions, I guess we can display the path in some cases when there are multiple links to distinguish between them :)

Thanks for all the work 💯

**Comment by anshumanv** [comment]
We should also update the repository description `A curated list of anime streaming/downloading websites.` as we have a bunch of other stuff too :)

**Comment by anshumanv** [comment]
@starkblaze01 need help with review

**Comment by toprak** [comment]
@anshumanv  Yes, we should but I do not know what to write :D

**Comment by anshumanv** [comment]
Thanks for the quick updates @toprak 🙌 

You can replace it with - 

> A curated list of all things anime - streaming/downloading/reading/tracking/forums

**Comment by anshumanv** [comment]
Also, since we have a bunch of sections can we have a `Categories` at the top of `Contents` and refer to all sections present?

**Comment by toprak** [comment]
@anshumanv I do not know github formatting I will add it midday.

**Comment by anshumanv** [comment]
Sure, anytime. Appreciate it! 🙌

**Comment by toprak** [comment]
@anshumanv done

File: README.md [hunk 4]
```diff
@@ -14,83 +14,260 @@ A curated list of anime streaming/downloading websites.
 * I am **NOT** promoting the use of free anime websites by any means.
 
 
-## Contents
+# Contents
+* [Anime](##Anime)
+ * [Streaming](###Streaming)
+ * [Downloading](###Downloading)
+  * [DDL](####DDL)
```
Comment (anshumanv) [inline README.md:21]: fix indentation, it appears as a separate point and not under `Downloading`

File: README.md [hunk 5]
```diff
@@ -14,83 +14,260 @@ A curated list of anime streaming/downloading websites.
 * I am **NOT** promoting the use of free anime websites by any means.
 
 
-## Contents
+# Contents
+* [Anime](##Anime)
```
Comment (anshumanv) [inline README.md:18]: Clicking on these links is not taking me to the correct section
```suggestion
* [Anime](#Anime)
```

This should work

**Review by anshumanv — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (toprak) [reply README.md:21]: I did not understand it :D

**Comment by anshumanv** [comment]
You can take help about the format from here - https://raw.githubusercontent.com/aviaryan/awesome-no-login-web-apps/master/README.md

File: README.md (hunk 4 above)
Comment (anshumanv) [reply README.md:21]: If you see it, it looks like this

image

DDL should be a sub-section under download right?

Comment (toprak) [reply README.md:21]: I fixed it wait im gonna commit it rn.

**Comment by toprak** [comment]
I updated it but they are not redirecting.. :thinking:

File: README.md [hunk 6]
```diff
@@ -14,83 +14,260 @@ A curated list of anime streaming/downloading websites.
 * I am **NOT** promoting the use of free anime websites by any means.
 
 
-## Contents
+# Contents
+* [Anime](#Anime)
+ * [Streaming](###Streaming)
```
Comment (anshumanv) [inline README.md:19]: you need to do this for all links, only need single `#`

```suggestion
 * [Streaming](#Streaming)
```

**Review by anshumanv — CHANGES_REQUESTED** [review]

**Comment by anshumanv** [comment]
> I updated it but they are not redirecting.. 🤔

Try clicking on `Anime`

**Comment by toprak** [comment]
it should be fine right now

File: README.md [hunk 7]
```diff
@@ -14,83 +14,260 @@ A curated list of anime streaming/downloading websites.
...
 
 
-## Contents
+# Contents
+* [Anime](#Anime)
+ * [Streaming](#Streaming)
+ * [Downloading](#Downloading)
+   * [DDL](#DDL)
+  * [Torrents](#Torrents)
+    * [Public Trackers](#PublicTrackers)
```
Comment (anshumanv) [inline README.md:23]: Same for all multiple words links

```suggestion
    * [Public Trackers](#Public-Trackers)
```

File: README.md [hunk 8]
```diff
@@ -14,83 +14,260 @@ A curated list of anime streaming/downloading websites.
...
+    * [Private Trackers](#PrivateTrackers)
+  * [Listing/Database](#Listing/Database)
+  * [Schedules/Charts](#Schedules/Charts)
+  * [Forums/Boards for discussing Anime](#Forums/BoardsfordiscussingAnime)
```
Comment (anshumanv) [inline README.md:27]: Just forums should be good 

```suggestion
  * [Forums](#Forums)
```

File: README.md [hunk 9]
```diff
@@ -14,83 +14,260 @@ A curated list of anime streaming/downloading websites.
...
+  * [Useful apps/programs about Anime.](#Usefulapps/programsaboutAnime.)
```
Comment (anshumanv) [inline README.md:28]: Make it just `Apps`

**Comment by toprak** [comment]
did it.

File: README.md [hunk 10]
```diff
@@ -14,83 +14,260 @@ A curated list of anime streaming/downloading websites.
 * I am **NOT** promoting the use of free anime websites by any means.
 
 
-## Contents
+# Contents
+* [Anime](#Anime)
+ * [Streaming](#Streaming)
```
Comment (anshumanv) [inline README.md:19]: Streaming is not coming as a subsection of anime, need to fix indentation I think

**Review by anshumanv — CHANGES_REQUESTED** [review]
Awesome! We're almost there :)

File: README.md (hunk 10 above)
Comment (toprak) [reply README.md]: try now

**Comment by toprak** [comment]
I guess its okay now

**Comment by anshumanv** [comment]
I think we're good now! Looks good, I'll merge after verifying all the links :)

**Comment by starkblaze01** [comment]
wow, that's a lot of work!

File: README.md [hunk 11]
```diff
@@ -14,83 +15,260 @@ A curated list of anime streaming/downloading websites.
...
+## Anime
+
+### Streaming
 * [9anime.to](https://www2.9anime.to/)
-* [anilinkz.to](http://anilinkz.to/)
+* [anime1.com](https://www.anime1.com/)
+* [anime8.ru](https://anime8.ru/)(Downloading and Streaming)
+* [animebam.se](https://www.animebam.se)
+* [animedaisuki.moe](https://animedaisuki.moe/)
+* [animedao.to](https://animedao.to/)
```
[the remark this answers is not in the document]
Comment (toprak) [reply README.md:51]: it works on me.

[the remark this answers is not in the document]
Comment (anshumanv) [reply README.md:51]: fails for me too :(

[the remark this answers is not in the document]
Comment (anshumanv) [reply README.md:51]: image

[the remark this answers is not in the document]
Comment (toprak) [reply README.md:51]: Screenshot_2020-09-18_09-21-17
It is still working on me :D

[the remark this answers is not in the document]
Comment (anshumanv) [reply README.md:51]: Yeah seems to work with vpn

**Comment by anshumanv** [comment]
@starkblaze01 good to merge?

**Review by anshumanv — APPROVED** [review]

**Comment by anshumanv** [comment]
Thanks for all the help @toprak!

This PR is shining ✨ 

I've invited you to the repository to collaborate 😄

**Comment by toprak** [comment]
Wow man, it has been 32 second since revolution is done, it feels so good :D

**Pull request merged**